sdks/python/apache_beam/examples/inference/runinference_metrics/pipeline/transformations.py

```diff
@@ -0,0 +1,93 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""This file contains the transformations and utility functions for
+the pipeline."""
+import apache_beam as beam
+import torch
+from apache_beam.io.filesystems import FileSystems
+from apache_beam.ml.inference.pytorch_inference import PytorchModelHandlerKeyedTensor
+from transformers import DistilBertForSequenceClassification
+from transformers import DistilBertTokenizer
+
+
+class CustomPytorchModelHandlerKeyedTensor(PytorchModelHandlerKeyedTensor):
+  """Wrapper around PytorchModelHandlerKeyedTensor to load a model on CPU."""
+  def load_model(self) -> torch.nn.Module:
+    """Loads and initializes a Pytorch model for processing."""
+    model = self._model_class(**self._model_params)
+    model.to(self._device)
+    file = FileSystems.open(self._state_dict_path, "rb")
+    model.load_state_dict(torch.load(file, map_location=self._device))
+    model.eval()
+    return model
+
+
+class HuggingFaceStripBatchingWrapper(DistilBertForSequenceClassification):
+  """Wrapper around HuggingFace model because RunInference requires a batch
+    as a list of dicts instead of a dict of lists. Another workaround
```

Comment:
Could you please create an issue to support this and then link to the issue in the code here? This is something we should probably be able to just infer/handle automatically